mailet/base/src/test/java/org/apache/mailet/base/test/FakeMail.java

```diff
@@ -39,6 +41,17 @@
 
 public class FakeMail implements Mail {
 
+    public static FakeMail fromMime(String text, String javaEncoding, String javamailEncoding) throws MessagingException, UnsupportedEncodingException {
```

Comment:
javamail has a property "mail.mime.charset" : `The mail.mime.charset System property can be used to specify the default MIME charset to use for encoded words and text parts that don't otherwise specify a charset. Normally, the default MIME charset is derived from the default Java charset, as specified in the file.encoding System property. Most applications will have no need to explicitly set the default MIME charset. In cases where the default MIME charset to be used for mail messages is different than the charset used for files stored on the system, this property should be set. `

It's not clear why it should introduce bugs but previous comments said so.

I'm not sure how to better encode this knowledge in the code, any idea ?